{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/templates/base.html

```diff
@@ -16,7 +16,9 @@
             <title>Lightmatter</title>
         {% endblock extra_head %}
     </head>
-    <body hx-boost="true"
+    <body x-boost="true"
+          hx-ext="get-csrf"
```

Comment:
Can you pull this line out? the csrf stuff should be it's own pr

Reply:
Fixed!